<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Cadastro</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bebas Neue Bold</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CLASSIFICADOR    DE   RECEITAS</string>
      </property>
     </widget>
     <widget class="QLabel" name="logotipo">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../logotipo classificador.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="plano_de_fundo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>-40</x>
        <y>0</y>
        <width>791</width>
        <height>491</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../plano de fundo_1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>112</y>
        <width>721</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Bebas Neue Regular</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Nome da receita</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="input_nome_receita"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>Bebas Neue Regular</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>LINK DA RECEITA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="input_link"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Bebas Neue Regular</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Grau de dificuldade</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="input_dificuldade">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fácil</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Médio</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Díficil</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Profissional</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Bebas Neue Regular</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Categoria</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="input_categoria">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Entrada</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Prato Principal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sobremesa</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Aperitivo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Acompanhamento</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lanche</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Salada</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Molho</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Bebas Neue Regular</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Estrelas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RB4">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RB5">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RB6">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RB7">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RB8">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="BT1_cadastro">
         <property name="font">
          <font>
           <family>Bebas Neue Regular</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cadastro</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Banco de Dados(Cadastros)</string>
     </attribute>
     <widget class="QLabel" name="plano_de_fundo_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>761</width>
        <height>491</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../plano de fundo_1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabela_dados">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>731</width>
        <height>341</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bebas Neue Regular</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>133</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Categoria</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Grau de dificuldade</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Estrelas</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Link</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Classificação</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>721</width>
        <height>66</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="BT_ordenarPorNome">
           <property name="font">
            <font>
             <family>Bebas Neue Regular</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Organizar por nome</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="OrganizarPelaDificuldade">
           <property name="font">
            <font>
             <family>Bebas Neue Regular</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Organizar pela dificuldade</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BT_organizarPelasEstrelas">
           <property name="font">
            <font>
             <family>Bebas Neue Regular</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Organizar pelas estrelas</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="ResetarDados">
           <property name="font">
            <font>
             <family>Bebas Neue Regular</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Resetar Tabela</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AtualizarDados">
           <property name="font">
            <font>
             <family>Bebas Neue Regular</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Atualizar Dados</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>721</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Bebas Neue Regular</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Número de receitas cadastradas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="numero_receitas"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Bebas Neue Regular</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Média de estrelas por receita</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="media_estrelas_geral"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
    <addaction name="actionExcluir"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionExcluir">
   <property name="text">
    <string>Excluir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
